claude/skills/wiremock-capture/SKILL.md

```
@@ -0,0 +1,152 @@
+- `/_apis/resourceAreas` - API discovery
+- `/_apis/wit/wiql` - WIQL queries
+- `/_apis/wit/workItems` - Work item operations
+- `/_apis/wit/workitemsbatch` - Batch operations
+- `/_apis/wit/workItemTypes` - Type definitions
+- `/_apis/wit/fields` - Field definitions
+- `/_apis/projects` - Project info
+
+### 5. Output Location
+
+Captured traffic goes to:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)